src/compiler/GF/Infra/SIO.hs

```diff
@@ -60,7 +60,7 @@ instance Monad SIO where
   SIO m1 >>= xm2 = SIO $ \ h -> m1 h >>= \ x -> unS (xm2 x) h
 
 instance Fail.MonadFail SIO where
-  fail = lift0 . fail
+  fail = liftSIO . fail
```

Comment:
Did you do your changes from an old commit? This was fixed in [1ff6600](https://github.com/GrammaticalFramework/gf-core/commit/1ff66006b8d928da01a56ca9f930f07b3cb2a01e).

Reply:
Yes, have git pulled the latest commits into my forked repo :)